translation.json

```diff
@@ -327,19 +335,19 @@
     "environmental_suit_description": "A bulky suit that protects the wearer from lack of oxygen, vacuum, radiation, and has maneuvering thrusters for use in microgravity.",
     "remote_vehicle_control": "Remote vehicle control",
     "remote_vehicle_control_description": "A tool allowing a pilot to remotely control a vehicle but it’s not as effective as being in the cockpit.",
-    "gatherinfo_what_is_their_vehicle_capable_of": "What is their vehicle capable of?",
-    "gatherinfo_when_are_their_vehicles_unattended": "When are their vehicles unattended?",
-    "gatherinfo_whats_the_best_way_through": "What's the best way through?",
-    "gatherinfo_where_did_x_go": "Where did [X] go?",
     "playbook_ace_description": "A graceful\nand daring\npilot",
     "playbook_ace_xp_condition": "You addressed a challenge with piloting or violence.",
+    "playbook_technician_description": "A savvy\ntechnician and\nvandal.",
+    "playbook_technician_xp_condition": "You addressed a challenge with technical skill or mayhem.",
     "playbook_item_grav_chute": "Grav chute",
     "playbook_item_fine_remote_vehicle_control": "Fine remote vehicle control",
     "playbook_item_self_destruct_remote": "Self destruct remote",
     "playbook_item_fine_environmental_suit_wearing": "Fine environmental suit (W)",
     "playbook_item_fine_environmental_suit_carried": "Fine environmental suit (C)",
     "playbook_item_your_custom_vehicle": "Your custom vehicle",
+    "playbook_item_fine_engineering_tools": "Expertly made tools used for bine mechanical work.",
```

Comment:
The name of the item (`playbook_item_fine_engineering_tools`) should just be its name: "Fine Engineering Tools"

To add a description, add one in the `playbookitem` key with the key `description` e.g.
```
"bold": "1",
"name": "playbook_item_fine_engineering_tools",
"description": "playbook_item_fine_engineering_tools_description",
"numboxes": "1"
```

Reply:
Thanks for the heads up--fixed these and done it right for the rest of the gear.